echo.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import (
+  "net/http"
+  "github.com/labstack/echo"
+)
+
+func Hello(c echo.Context) error {
```

Comment:
sebetulnya opsional tapi apa bisa ditambahkan komentar di sebelum function Hello ya mas?
kalau tanpa komentar, ada warning seperti gambar di bawah:

![warning function Hello](https://cdn.discordapp.com/attachments/750304684836651039/813252273354178590/unknown.png)

komentarnya biasanya formatnya nama function terus keterangan nya, contohnya seperti di bawah

![komentar sebelum function](https://cdn.discordapp.com/attachments/750304684836651039/813252674321383464/unknown.png)


```suggestion
// Hello menginformasikan bahwa API berjalan dengan baik
func Hello(c echo.Context) error {
```